events/release-event-action/action.yml

```diff
@@ -25,7 +25,11 @@ inputs:
    JDK_VERSION:
      required: false
      description: "The version of the JDK to setup."
-     default: 11
+     default: 17
+  IGNORE_TEST_FAILURES:
+    required: false
+    description: "Allow tests to fail."
+    default: false
```

Comment:
I think this is the only one where we should force it not to ignore them, as it concerns a release.

Reply:
Myeah, probably a good idea. Can always disable the test if you get really desperate when releasing.